python/src/CrossEntropyResult_doc.i.in

```diff
@@ -0,0 +1,70 @@
+%feature("docstring") OT::CrossEntropyResult
```

Comment:
The warning about the experimental module is missing from this doc.

Reply:
You're true, thanks.